src/app/app.js

```diff
@@ -242,6 +242,10 @@ win.on('enter-fullscreen', function () {
 
 // Wipe the tmpFolder when closing the app (this frees up disk space)
 win.on('close', function () {
+    close();
+});
+// Now this function is used via global keys (cmd+q and alt+f4)
+function close() {
```

Comment:
To pass the CI and make the linter happy, you need to declare the function before you call it.